rust/Cargo.toml

```diff
@@ -25,5 +27,5 @@ crate-type = ["staticlib"]
 
 [profile.release]
 panic = "abort"
-lto = true
+#lto = true
```

Comment:
Also this...um...yeah LTO is slow.  Let's default it to off and only do it for production RPM builds or something?

Reply:
That sounds fine to me, yeah. (How much slower was it roughly BTW?)